setup.py

```diff
@@ -17,13 +17,16 @@ def read_relative_file(filename):
         description="Let you be behind a nginx configuration an get "
                     "x509 auth credentials",
         long_description=read_relative_file('README.rst'),
-        classifiers=['Development Status :: 4 - Beta',
-                     'Environment :: Web Environment',
-                     'Intended Audience :: Developers',
-                     'License :: OSI Approved :: MIT License',
-                     'Programming Language :: Python',
-                     'Programming Language :: Python :: 2.7',
-                     'Programming Language :: Python :: 3'],
+        classifiers=[
+            'Development Status :: 5 - Production/Stable',
+            'Environment :: Web Environment',
+            'Intended Audience :: Developers',
+            'License :: OSI Approved :: MIT License',
+            'Programming Language :: Python',
+            'Programming Language :: Python :: 2.7',
```

Comment:
Do we need to add `'Programming Language :: Python :: 2',` prior to 2.7? Not sure.